doc/manual/queryformat

```diff
@@ -90,11 +89,11 @@ grep or awk). If you try the obvious,
     rpm -q --queryformat "[%{NAME} %{FILENAMES}\n]" cdp
```

Comment:
While we're updating package names, here's another candidate. Even I don't remember what "cdp" used to do :D